src/lib/write-package.ts

```diff
@@ -150,21 +150,24 @@ export const writePackage = async (
   // make sure publish user has permission to publish the package
   // get the github repository from packument
   if (!repo) {
+    console.info('failed to find repository in latest.repository field.')
     res.status(400);
     const ret = {
       error: formatError(
         'in order to publish the latest version must have a repository on github'
       ),
       statusCode: 400,
     };
-    res.end(ret.error);
+    res.end(JSON.stringify(ret));
     return ret;
   }
 
   let repoResp = null;
   try {
     repoResp = await github.getRepo(repo.name, user.token);
   } catch (e) {
+    console.info('failed to get repo response for '+repo.name+' '+e)
```

Comment:
Thoughts on using backticks and interpolation?

Reply:
I'm :man_shrugging: 
if you don't have new lines in your message back ticks are nearly as easy.